src/source/Ice/TurnConnection.c

```diff
@@ -1033,6 +1033,8 @@ STATUS checkTurnPeerConnections(PTurnConnection pTurnConnection)
     PStunAttributeChannelNumber pStunAttributeChannelNumber = NULL;
     UINT32 i = 0;
 
+    UNUSED_PARAM(sendStatus);
```

Comment:
Can we add a check log error for this line?

The value is not being checked below.

Reply:
Agree, we should log this.